nixos/modules/services/security/authelia.nix

```diff
@@ -379,6 +380,15 @@ in
               Do not include raw secrets in nix settings.
             '';
           }
+          # Removed options. Cannot use mkRemovedOptionModule (or similar methods) due to errors.
+          # See https://github.com/NixOS/nixpkgs/pull/299309#issuecomment-2038240493
+          {
+            assertion = !((instance.settings.server ? host) || (instance.settings.server ? port));
```

Comment:
Haven't tested this, but wouldn't the `host` and `port` options have the same issue of never getting to this assertion ....

Reply:
`host` and `port` do not have the same problem: if I try to set `settings.server.host = "foo"`, I get the assertion error.

I think this is because `host` and `port` are part of the `settings` option which allows setting even values that are not defined in the module (as the entire `settings` value is then formatted as the main YAML config file).

Reply:
Ah, ok, got it.